tools/Cron_jobs/mailwatch_geoip_update.php

```diff
@@ -170,7 +170,7 @@
 
         // Apply MailWatch rights on files from the last run
         $mwUID =  exec('cat /etc/sudoers.d/mailwatch | grep "User_Alias MAILSCANNER" | sed "s/.*= \(.*\).*/\1/"', $retval_cat);
-        if ($retval_cat > 0) {
```

Comment:
ok, the second parameter is missing ($output) which is an array, $retval_cat must be the third